test/evaluate/testing.cc

```diff
@@ -108,8 +107,8 @@ FailureDetailPrinter Compare(const char *file, int line, const char *xs,
     return BitBucket;
   } else {
     ++failures;
-    fprintf(stderr, "%s:%d: FAIL: %s[0x%llx] %s %s[0x%llx]\n", file, line, xs,
-        x, rel, ys, y);
+    fprintf(stderr, "%s:%d: FAIL: %s[0x%jx] %s %s[0x%jx]\n", file, line, xs,
+        static_cast<std::uint64_t>(x), rel, ys, static_cast<std::uint64_t>(y));
```

Comment:
These casts should be to `std::uintmax_t` as above.

Reply:
Thanks!